tsuru/client/auth_test.go

```diff
@@ -602,7 +603,7 @@ func (s *S) TestShowAPITokenRunWithFlag(c *check.C) {
 		CondFunc: func(req *http.Request) bool {
 			called = true
 			return req.Method == "GET" && strings.HasSuffix(req.URL.Path, "/users/api-key") &&
-				req.URL.RawQuery == "user=admin@example.com"
+				req.URL.RawQuery == "user=" + url.QueryEscape("admin@example.com")
```

Comment:
This line is broken the Go format. You can use the `gofmt` command to check and fix the format problems: https://golang.org/cmd/gofmt/

In this case the `gofmt` will remove the space between the string and the `+` operator